clients/githubrepo/repo.go

```diff
@@ -83,8 +88,10 @@ func (r *repoURL) String() string {
 
 // IsValid implements Repo.IsValid.
 func (r *repoURL) IsValid() error {
+	githubHost := os.Getenv("GH_HOST")
```

Comment:
Would a user be setting the GH_HOST field twice? Could they instead pass in `--repo-url` and we use the environment variable to determine whether GH Enterprise is being used, or not?

Reply:
It is intended to setup - GH_HOST only once as environment variable similar to GITHUB_TOKEN. Though, consumer might configure github host twice (env and repo full URL). We can clarify this in documentation if that helps. In short, we only need one and env version fit their.

`GH_HOST` is one of [github cli environment](https://cli.github.com/manual/gh_help_environment) based approach to switch GHES and provide convenience.

Technically we can use without env version and push consumer to use full URL to identify whether it is GHEC or GHES, though Gitlab in mix can add conditional check. IDK if GitLab also have its Cloud and Enterprise version. With those uncertainty I see GH_HOST as env fit well.

Though I'm open on either way.

Reply:
Gotcha, I wasn't aware that it was also a GitHub CLI environment variable. That makes sense.